src/WinRT.Runtime/Projections/Nullable.cs

```diff
@@ -2203,6 +2006,139 @@ internal static class Nullable_Delegates
+            if (type == typeof(short)) return NullableBlittable<short>.GetValue;
+            if (type == typeof(ushort)) return NullableBlittable<ushort>.GetValue;
+            if (type == typeof(char)) return NullableBlittable<char>.GetValue;
+            if (type == typeof(uint)) return NullableBlittable<uint>.GetValue;
+            if (type == typeof(long)) return NullableBlittable<long>.GetValue;
+            if (type == typeof(ulong)) return NullableBlittable<ulong>.GetValue;
+            if (type == typeof(float)) return NullableBlittable<float>.GetValue;
+            if (type == typeof(double)) return NullableBlittable<double>.GetValue;
+            if (type == typeof(Guid)) return NullableBlittable<Guid>.GetValue;
+            if (type == typeof(global::System.Type)) return Nullable_Type.GetValue;
+            if (type == typeof(global::System.TimeSpan)) return Nullable_TimeSpan.GetValue;
+            if (type == typeof(global::System.Exception)) return Nullable_Exception.GetValue;
+            if (type == typeof(global::System.DateTimeOffset)) return Nullable_DateTimeOffset.GetValue;
+            if (type == typeof(global::Windows.Foundation.Point)) return NullableBlittable<global::Windows.Foundation.Point>.GetValue;
+            if (type == typeof(global::Windows.Foundation.Size)) return NullableBlittable<global::Windows.Foundation.Size>.GetValue;
+            if (type == typeof(global::Windows.Foundation.Rect)) return NullableBlittable<global::Windows.Foundation.Rect>.GetValue;
+            if (type == typeof(global::System.Numerics.Matrix3x2)) return NullableBlittable<global::System.Numerics.Matrix3x2>.GetValue;
+            if (type == typeof(global::System.Numerics.Matrix4x4)) return NullableBlittable<global::System.Numerics.Matrix4x4>.GetValue;
+            if (type == typeof(global::System.Numerics.Plane)) return NullableBlittable<global::System.Numerics.Plane>.GetValue;
+            if (type == typeof(global::System.Numerics.Quaternion)) return NullableBlittable<global::System.Numerics.Quaternion>.GetValue;
+            if (type == typeof(global::System.Numerics.Vector2)) return NullableBlittable<global::System.Numerics.Vector2>.GetValue;
+            if (type == typeof(global::System.Numerics.Vector3)) return NullableBlittable<global::System.Numerics.Vector3>.GetValue;
+            if (type == typeof(global::System.Numerics.Vector4)) return NullableBlittable<global::System.Numerics.Vector4>.GetValue;
+            if (type == typeof(global::System.EventHandler)) return Nullable_EventHandler.GetValue;
+            if (type.IsEnum && Enum.GetUnderlyingType(type) == typeof(int)) return (inspectable) => Nullable_IntEnum.GetValue(type, inspectable);
+            if (type.IsEnum && Enum.GetUnderlyingType(type) == typeof(uint)) return (inspectable) => Nullable_FlagsEnum.GetValue(type, inspectable);
+
+#if NET
+            var winrtExposedClassAttribute = type.GetCustomAttribute<WinRTExposedTypeAttribute>(false);

```

Comment:
To reduce code size (after making this method generic), we could just outline this fallback code to a non-generic method that just takes a `Type` instance, that'd be fine, since the type isn't statically known at this point anyway.

Reply:
Connected to the other feedback where this function can't be generic due to how it is used right now.